lib/web_ui/dev/canvaskit_lock.yaml

```diff
@@ -1,4 +1,4 @@
 # Specifies the version of CanvasKit to use for Flutter Web apps.
 #
 # See `lib/web_ui/README.md` for how to update this file.
-canvaskit_version: "0.35.0"
+canvaskit_version: "0.36.1"
```

Comment:
What are your thoughts on rolling to 0.37.0? https://www.npmjs.com/package/canvaskit-wasm/v/0.37.0

Or is the idea to do the minimal update to minimize potential breaks?

Reply:
Yeah, I started a draft - https://github.com/flutter/engine/pull/36011 - but I want to isolate the golden flakiness we're seeing into separate rolls. I'm not sure if 0.37.0 could contain other surprises, such as unfixing the issue that 0.36.1 is fixing or some such.

Reply:
Makes sense! Marking as resolved